### PR TITLE
Document that offsets are returned in logical pixels

### PR DESCRIPTION
**Pull request by goderbauer**
## Description

Make documentation more specific to avoid user confusion.

## Related Issues

None

## Tests

I added the following tests:

None, doc only change.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Review by Piinks — APPROVED** [review]
LGTM! Thanks for making this clearer. :)

File: packages/flutter_driver/lib/src/common/geometry.dart [hunk 1]
```diff
@@ -29,6 +29,9 @@ EnumIndex<OffsetType> _offsetTypeIndex = EnumIndex<OffsetType>(OffsetType.values
 
 /// A Flutter Driver command that return the [offsetType] from the RenderObject
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/common/geometry.dart]: returns

File: packages/flutter_driver/lib/src/common/geometry.dart [hunk 2]
```diff
@@ -29,6 +29,9 @@ EnumIndex<OffsetType> _offsetTypeIndex = EnumIndex<OffsetType>(OffsetType.values
...
 /// identified by [finder].
+///
+/// The requested offset is returned in logical pixels, which can be translated
+/// to device pixels via the [Window.devicePixelRatio].
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/common/geometry.dart]: s/the //

File: packages/flutter_driver/lib/src/common/geometry.dart [hunk 3]
```diff
@@ -51,14 +54,23 @@ class GetOffset extends CommandWithTarget {
 }
 
 /// The result of the [GetRect] command.
+///
+/// The offset is provided in logical pixels, which can be translated
+/// to device pixels via the [Window.devicePixelRatio].
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/common/geometry.dart]: s/the //

File: packages/flutter_driver/lib/src/common/geometry.dart [hunk 4]
```diff
@@ -51,14 +54,23 @@ class GetOffset extends CommandWithTarget {
...
 class GetOffsetResult extends Result {
   /// Creates a result with the offset defined by [dx] and [dy].
   const GetOffsetResult({ this.dx = 0.0, this.dy = 0.0});
 
-  /// The x component of the offset.
+  /// The x component of the offset in logical pixels.
+  ///
+  /// The value can be translated to device pixels via the
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/common/geometry.dart]: s/ the//

File: packages/flutter_driver/lib/src/common/geometry.dart [hunk 5]
```diff
@@ -51,14 +54,23 @@ class GetOffset extends CommandWithTarget {
...
+  /// [Window.devicePixelRatio].
   final double dx;
 
-  /// The y component of the offset.
+  /// The y component of the offset in logical pixels.
+  ///
+  /// The value can be translated to device pixels via the
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/common/geometry.dart]: s/ the//

File: packages/flutter_driver/lib/src/driver/driver.dart [hunk 6]
```diff
@@ -483,26 +483,41 @@ class FlutterDriver {
   }
 
   /// Returns the point at the top left of the widget identified by `finder`.
+  ///
+  /// The offset is expressed in logical pixels and can be translated to
+  /// device pixels via the [Window.devicePixelRatio].
```
Comment (tvolkert) [inline packages/flutter_driver/lib/src/driver/driver.dart]: s/ the//

(and below)

**Review by tvolkert — APPROVED** [review]
LGTM

**Pull request merged**